Bootstrap v4 - (Breadcrumbs) Breadcrumb.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Breadcrumb",
+    "heading": "breadcrumb",
+    "description": "Indicate the current page's location within a navigational hierarchy that automatically adds separators via CSS.",
+    "usage": "Separators are automatically added in CSS through ::before and content.",
+    "snippet": "<ol class=\"\"breadcrumb\"\">\r\n  <li class=\"\"breadcrumb-item active\"\">Home<\/li>\r\n<\/ol>\r\n<ol class=\"\"breadcrumb\"\">\r\n  <li class=\"\"breadcrumb-item\"\"><a href=\"\"#!\"\">Home<\/a><\/li>\r\n  <li class=\"\"breadcrumb-item active\"\">Library<\/li>\r\n<\/ol>\r\n<ol class=\"\"breadcrumb\"\">\r\n  <li class=\"\"breadcrumb-item\"\"><a href=\"\"#!\"\">Home<\/a><\/li>\r\n  <li class=\"\"breadcrumb-item\"\"><a href=\"\"#!\"\">Library<\/a><\/li>\r\n  <li class=\"\"breadcrumb-item active\"\">Data<\/li>\r\n<\/ol>",
```

Comment:
Here is some more of those :(

Reply:
Hmm. I just edited that one, again, and it is the only one in the batch that I see that has that issue.

Reply:
I have seen more, I marked a few...